docs/user-guides/creating-a-simple-ca-based-issuer.md

```diff
@@ -0,0 +1,53 @@
+
+We are going to create an `Issuer` that will use this CA to generate signed certificates. You can read more about the `Issuer` resource [here][3]. To allow the `Issuer` to reference our CA we need to put it into a `Secret`. `Issuers` are namespaced resources and can only reference and create `Secrets` in their own namespace, so we will need to make sure to put our CA `Secret` into the same namespace as our `Issuer`. We could alternatively create a `ClusterIssuer` in which case our `Secret` will need to be in the namespace specified by the `--cluster-resource-namespace` flag on the `cert-manager` controller.
+
+The following command will create a `Secret` containing our CA in the default namespace.
+
+```bash
+$ kubectl create secret tls ca-key-pair --cert=/path/to/ca.crt --key=/path/to/ca.key --namespace default
+```
+
+We can now create an `Issuer` referencing our `Secret`.
+
+```yaml
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Issuer
+metadata:
+  name: ca-issuer
+  namespace: default
+spec:
+  ca:
+    secretName: ca-key-pair
+```
+
+We are now ready to obtain certificates. We can create the following `Certificate` resource which specifies our desired certificate. You can read more about the `Certificate` resource [here][4]. Note that to use the `Issuer` above to obtain our certificate, we must create the `Certificate` resource in the same namespace as the `Issuer`.
+
+```yaml
+kind: Certificate
+metadata:
+  name: ca-crt
+  namespace: default
+spec:
+  secretName: ca-crt-secret
+  issuerRef:
+    name: ca-issuer
+    # We can reference ClusterIssuers by changing the kind here.
+    # The default value is Issuer (i.e. a locally namespaced Issuer)
+    kind: Issuer
+  dnsNames:
+  - cert-manager.k8s.io
+```
+
+Once we have created the `Certificate` resource, `cert-manager` will attempt to use our `ca-issuer` to obtain a certificate. If successful, the certificate will be put into a `Secret` called `ca-crt-secret` in the default namespace. Note that since we have not specified the `commonName` on the `Certificate`, the first element of the `dnsNames` array will be used as the common name and any other elements would be [Subject Alternative Names][5] (SANs). If we had specified the `commonName`, then all the elements of the `dnsNames` array would be SANs.
+
+Once our certificate has been obtained, `cert-manager` will keep checking its validity and attempt to renew it if it gets close to expiry. `cert-manager` considers certificates to be close to expiry when the 'Not After' field on the certificate is less than the current time plus 30 days. For CA `Issuers`, `cert-manager` will issue certificates with the 'Not After' field set to the current time plus 365 days. To learn more about the 'Not After' field, look at the [structure of an X.509 certificate][6].
```

Comment:
`To learn more about the 'Not After' field, look at the [structure of an X.509 certificate][6]` - don't think there's much value linking to specific details on the 'not after' field. Unless there's something specifically interesting/relevant/important for a user to take account of?

Reply:
Yeah there isn't really